extension/src/experiments/commands/register.ts

```diff
@@ -318,19 +320,22 @@ const registerExperimentRunCommands = (
   internalCommands.registerExternalCommand(
     RegisteredCommands.EXPERIMENT_SHOW,
     (context: Context) =>
-      experiments.showWebview(getDvcRootFromContext(context))
+      setup.shouldBeShown()
+        ? setup.showWebview()
```

Comment:
Had a thought, maybe we should use `commands.executeCommand` here so that we get a telemetry event for the redirect.